### ISSUE TITLE
Error: The settings file could not be opened

### ISSUE THREAD

**Issue by raineorshine** [yaroslavyaroslav/OpenAI-sublime-text#5]
## Environment

- OSX 13.1
- Sublime Text Build 4143

## Steps to Reproduce

1. Install OpenAI-sublime-text from package control
	- Added to `/Installed Packages/OpenAI completion.sublime-package`
2. Sublime Text → Settings... → Package Settings → OpenAI completion → Settings

## Current Behavior

```
The settings file "res://Packages/openAI/openAI.sublime-settings" could not be opened
```

- There is no `openAI` directory in `/Packages`. 
- An error also occurs if "Key Bindings" is chosen from the menu.
- When I start Sublime Text, there are no errors in the console.

## Expected Behavior

User settings should be opened.

**Comment by deathaxe** [comment]
Main menu uses the wrong package name `openAI`. A package name must match the install name of the package, which is `OpenAI completion`. 

https://github.com/yaroslavyaroslav/OpenAI-sublime-text/blob/09dafc198716ea144c42dde477e3ed9847f1ac64/Main.sublime-menu#L19

**Comment by yaroslavyaroslav** [comment]
@deathaxe is there spaces allowed within a name? Like should i rename it directly to as you said above: `OpenAI completion.sublime-settings`?

**Comment by deathaxe** [comment]
> is there spaces allowed within a name?

Package names can contain spaces. The only limitation is that plugins located within such packages can't be imported by other packages, but thats a rather rare use case with other implications anyway.

> Like should i rename it directly to as you said above: OpenAI completion.sublime-settings?

At least the package nme must be renamed to fix the issue. Whether the settings file should be renamed as well, is a rather strategic question. It is a common design pattern to use the package name for settings files as well, just to avoid any confusion.

**Comment by mattlinares** [comment]
I take it this still needs to be submitted to the Sublime package repo or something right?

**Comment by raineorshine** [comment]
@mattlinares It's been published at `v1.1.3`.
